src/lib/Liblog/pbs_log.c

```diff
@@ -947,6 +956,13 @@ log_record(int eventtype, int objclass, int sev, const char *objname, const char
 			fclose(logfile);
 		}
 	}
+
+sigunblock:
+#ifndef WIN32
+	sigprocmask(SIG_SETMASK, &old_mask, NULL);
+#else
+	return;
```

Comment:
Shouldn't have to call return as the last line of a void function. Otherwise, looks good.